tests/processor/document/test_candidate_relation_adder.py

```diff
@@ -0,0 +1,246 @@
+import copy
+
+import pytest
+from pytorch_ie.annotations import BinaryRelation, LabeledSpan
+
+from pie_utils.processor.document import DocumentWithEntitiesRelationsAndPartition
+from pie_utils.processor.document.candidate_relation_adder import CandidateRelationAdder
+
+TEXT_01 = "Jane lives in Berlin. this is no sentence about Karl\n"
```

Comment:
Let's rename:
* `TEXT_01` to `TEXT1` etc
* `ENTITY_TEXT_01_JANE` to `ENTITY_JANE_TEXT1` etc
* `TEXT_01_SENTENCE1` to `SENTENCE1_TEXT1` etc